std/file.d

```diff
@@ -2927,7 +2929,7 @@ if (isConvertibleToString!R)
     dir.buildPath("a").write(".");
     dir.chdir; // step into dir
     "b".write(".");
-    dirEntries(".", SpanMode.shallow).equal(
+    dirEntries(".", SpanMode.shallow).array.sort.equal(
         [".".buildPath("b"), ".".buildPath("a")]
```

Comment:
```suggestion
        [".".buildPath("a"), ".".buildPath("b")]
```
?

Reply:
Ooof, this is weird. It makes sense the change actually, but it compiles in my machine and I didn't notice that. Maybe my brain is completely fried :upside_down_face:

Reply:
It will compile either way. Do you mean that it runs (and the assert passes)?

Reply:
> It will compile either way. Do you mean that it runs (and the assert passes)?

I got the point now. I was not crazy. There's actually no testing there, no `assert()` around the equal. :facepalm:

Reply:
Ah, whoops!

Reply:
Yep, just do `assert(dirEntries(".", SpanMode.shallow) == [".".buildPath("a"), ".".buildPath("b")])` and dlang.org should do the rest

Reply:
@ljmf00 Would you mind fixing this too please? (Fix order and add `assert`)

Previously it was maybe-sometimes-working, now this looks like it never works.

Reply:
Yes, I'm sorry, I forgot it. Should be good now